addons/pinniped/config-controller/hack/ytt/default-package-secret.yaml

```diff
@@ -0,0 +1,22 @@
+#@ load("@ytt:yaml", "yaml")
+#@ load("@ytt:data", "data")
+#@overlay/match-child-defaults missing_ok=True
+
+#@ def config():
+---
+infrastructure_provider: #@ data.values.infrastructure_provider
```

Comment:
Do we need these headers here?
```
    #@data/values
    #@overlay/match-child-defaults missing_ok=True
```

Reply:
So we don't seem to need them in that the script works, but I'll look at this quick.

Reply:
I'm leaning towards not including unless necessary.  Thoughts?

Reply:
I think we have resolved this.  We don't need them due to the use of `valuesFrom`.